tests/integration/suite/operator/api/listcomponents.go

```diff
@@ -88,6 +89,18 @@ func (l *listcomponents) Setup(t *testing.T) []framework.Option {
 		},
 	}
 
+	l.ctrlPlaneComp = &compapi.Component{
+		TypeMeta:   metav1.TypeMeta{Kind: "Component", APIVersion: "dapr.io/v1alpha1"},
+		ObjectMeta: metav1.ObjectMeta{Name: "ctrl-plane-comp", Namespace: "default"},
+		Spec: compapi.ComponentSpec{
+			Type:    "state.inmemory",
+			Version: "v1",
+		},
+		Scoped: common.Scoped{
+			Scopes: []string{"dapr-placement"},
```

Comment:
What happens when I include app IDs here?

Reply:
They will be ignored - Want to keep components for Apps and CP separate.